### ISSUE TITLE
image's alt text is missing 

### ISSUE THREAD

**Issue by s-kris** [mkhstar/suneditor-react#161]
Hello!

When uploading an image, we can attach alt text which is getting saved to database.
But when the post is loaded from database into suneditor-react, the alt text is missing. 

I tested this on both 2.14.10 and 2.15.1

**Comment by s-kris** [comment]
Not sure if this is caused by react wrapper, hence linking both: https://github.com/JiHong88/SunEditor/issues/601

**Comment by s-kris** [comment]
This is the fix: https://github.com/JiHong88/SunEditor/issues/601#issuecomment-765144010
it'll be fixed in editor-core in next version.
